src/admin/cli.py

```
@@ -238,6 +240,123 @@ def bridge_logout():
+
+    click.echo(f"Loading config: {config_path} ({len(cfg.communities)} community)")
+    click.echo("Reading Matrix state...\n")
+
+    report = reconciler.apply()
+
+    for action in report.actions:
+        icon = {"create": "+", "skip": "=", "invite": ">", "bridge": "~", "config": "*", "adopt": "!", "error": "X"}
+        click.echo(f"  [{icon.get(action.operation, '?')}] {action.resource:<28} {action.operation:<8} {action.details}")
+
+    if report.errors:
+        click.echo(f"\nErrors:", err=True)
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)